sources/packages/backend/package.json

```diff
@@ -72,69 +72,69 @@
     "bull": "^4.16.5",
     "clamscan": "^2.4.0",
     "class-transformer": "^0.5.1",
-    "class-validator": "^0.14.2",
+    "class-validator": "^0.14.3",
     "cookie-parser": "^1.4.7",
     "cron-parser": "^5.4.0",
-    "dayjs": "^1.11.18",
+    "dayjs": "^1.11.19",
     "dotenv": "^17.2.3",
     "express-basic-auth": "^1.2.1",
     "helmet": "^8.1.0",
-    "ioredis": "^5.8.1",
+    "ioredis": "^5.8.2",
     "jsonata": "^2.1.0",
     "papaparse": "^5.5.3",
     "passport-jwt": "^4.0.1",
     "pg": "^8.16.3",
     "prom-client": "^15.1.3",
     "qs": "^6.14.0",
-    "redis": "^5.8.3",
+    "redis": "^5.10.0",
     "reflect-metadata": "^0.2.2",
     "rxjs": "^7.8.2",
-    "soap": "^1.5.0",
+    "soap": "^1.6.1",
     "ssh2-sftp-client": "^12.0.1",
     "swagger-ui-express": "^5.0.1",
     "tslib": "^2.8.1",
     "typeorm": "^0.3.28",
     "uuid": "^13.0.0"
   },
   "devDependencies": {
-    "@aws-sdk/types": "^3.936.0",
+    "@aws-sdk/types": "^3.953.0",
```

Comment:
im surprised to see the aws sdk here!

Reply:
It is used to connect to the Gov S3 storage.